### ISSUE TITLE
Pluto writes to the notebook source file when `auto_reload_from_file=true`

### ISSUE THREAD

**Issue by rikhuijzer** [fonsp/Pluto.jl#2183]
Pluto writes to notebook source file after loading it with `auto_reload_from_file`. This causes editors to throw warnings such as

```
WARNING: The file has been changed since reading it
```

This shouldn't happen. There is no need for Pluto to write to the file if it is detecting updates.

Linux 64 bit with Pluto 0.19.9

EDIT: Happens with and without the built-in package manager
EDIT2: Note that I'm talking about Pluto making changes in the source file even if not touching anything in Pluto

**Comment by j-fu** [comment]
I think this should happen  - good editors like emacs have "auto-revert-mode"  (vim has something similar).  So I use this as a feature when developing notebooks in parallel in my favorite editor and in Pluto. 

E.g. just think about how to  add a new cell to the notebook. 

What indeed may be the case is that this happens too often.

**Comment by j-fu** [comment]
On another note, it indeed would be nice to have a switch preventing Pluto from writing the notebook file, independent of auto_reload_from_file  - see e.g. this discussion: 
https://julialang.zulipchat.com/#narrow/stream/243342-pluto.2Ejl/topic/.E2.9C.94.20Pluto.20and.20Git

**Comment by rikhuijzer** [comment]
> On another note, it indeed would be nice to have a switch preventing Pluto from writing the notebook file

Globally, there is already `disable_writing_notebook_files`. The Zulip link doesn't work for me

**Comment by rikhuijzer** [comment]
My issue here is that Pluto writes to the source file even if I didn't change anything in Pluto!

**Comment by j-fu** [comment]
This is indeed strange.  I agree that there should be no writing to disk without a change in the notebook.

**Comment by Pangoraw** [comment]
> WARNING: The file has been changed since reading it

You can use `:set autoread` to have two way synchronisation in vim.

**Comment by rikhuijzer** [comment]
> > WARNING: The file has been changed since reading it
> 
> You can use `:set autoread` to have two way synchronisation in vim.

@Pangoraw What don't you know :exploding_head:? I'll try that and close this issue for now
